src/screens/poa-form/styles.scss

```diff
@@ -47,9 +47,27 @@
   display: none !important;
 }
 
-.stepper div div div div { // HACK: Since the slider sets all styles inline.
-  a,
-  span {
-    line-height: 32px !important;
+.grommetux-box--direction-row.grommetux-box--responsive.button-box {
+  @media (max-width: 719px) {
+    flex-direction: column-reverse;
+    .button {
```

Comment:
I think you could do this less hackily by wrapping the buttons in a [Grommet  Box](http://grommet.io/docs/box/).  Can you try making that work?

Reply:
It is wrapped in a grommet box- but in classic opinionated-but-not-opinionated-enough fashion, it doesn't really go the right route. imho, next should be at the top on mobile, which would warrant a `column-reverse` and full-width buttons. I don't think its _horrible_ without the hack, and am happy to remove if we want to keep it clean.